src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17010.cs

```diff
@@ -0,0 +1,27 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue17010 : _IssuesUITest
+{
+	public Issue17010(TestDevice device) : base(device)
+	{
+	}
+
+	public override string Issue => "Unable to Update iOS SwipeGesture Direction at Runtime";
+
+	[Test]
+	[Category(UITestCategories.Gestures)]
+	public void SwipeGestureDirectionShouldChangeAtRuntime()
```

Comment:
The test is failing on Catalyst:
```
Assert.That(App.WaitForElement("DirectionLabel").GetText(), Is.EqualTo("Direction: Left"))
Expected string length 15 but was 16. Strings differ at index 11.
Expected: "Direction: Left"
But was:  "Direction: Right"
----------------------^
```

Reply:
@jsuarezruiz, I have restricted the test case for Mac because swipe actions are not supported in Appium on MacCatalyst. Please let me know if you have any concerns.

Reply:
Mnnn, there is a specific implementation using the Appium Mac Driver to execute swipe actions https://github.com/dotnet/maui/blob/4dc9f393acbe8a697a850aa99d3fbfeab36964e0/src/TestUtils/src/UITest.Appium/Actions/AppiumCatalystSwipeActions.cs
What is the current behavior? Does nothing, wrong swipe direction or other issue?

Reply:
Thanks for the feedback, should be fixed by https://github.com/dotnet/maui/pull/30436

Reply:
@jsuarezruiz, I have tested the test case using the code provided in this PR ([#30436](https://github.com/dotnet/maui/pull/30436)). The swipe action on Mac is now working as expected after applying the changes.